### ISSUE TITLE
Packages Viewing a package that has derived packages

### ISSUE THREAD

**Issue by Ana06** [openSUSE/open-build-service#5900]
Failing all the time:

```
  1) Packages Viewing a package that has derived packages
     Failure/Error: expect(page.current_path).to eq(package_show_path(project: branched_project, package: branched_project.packages.first))
     
       expected: "/package/show/home:package_test_user:branches:home:package_test_user/test_package"
            got: "/package/show/home:package_test_user/test_package"
     
       (compared using ==)
     # ./spec/features/webui/packages_spec.rb:53:in `block (3 levels) in <top (required)>'

```

**Comment by ChrisBr** [comment]
Is it really flickering when it fails everywhere?

**Comment by Ana06** [comment]
@ChrisBr 

> Is it really flickering when it fails everywhere?

I managed that it passes by triggering it several times... check [master](https://github.com/openSUSE/open-build-service/commits/master), sometimes it passes. I think it is flickering, just too much! It passes also locally.

**Comment by ChrisBr** [comment]
>  It passes also locally.

With bootstrap enabled (``export BOOTSTRAP=1``)?

**Comment by coolo** [comment]
Yes, it's always tab 3

**Comment by coolo** [comment]
Fixed by acb7208b53a95db08ad9e0c96859e9c44920c7b0 I suppose
